02-basics.Rmd

```diff
@@ -260,7 +260,7 @@ as.matrix(crashes)
 
 Often it is useful to 'recode' values.
 In the raw STATS19 files, for example, -1 means NA.
-There are many ways to recode values in R, the simplest and most mature of which is the use of factors, as shown below:
+There are many ways to recode values in R, the simplest and most mature of which is the use of factors DEFINE WHAT A FACTOR IS, as shown below:
```

Comment:
How about:

```
 There are many ways to recode values in R, the simplest and most mature of which is the use of factors, which are whole numbers representing characters, as shown below: 
```

On second thoughts, is this even a useful example? It is a bit confusing I think so it may be worth scrapping it IMO...

Reply:
There are quite a few use cases for using factors in STATS19 eg recoding vehicles to simpler categories so I think it is helpful to have a look at factors.  I think the wording you have suggested is better - perhaps worth saying something that factors are usually used for categorical data - i think that would help - would a male/femae/other example would be more helpful?

Reply:
I think the wording is now good.  A better example might be to use gender e.g male, female, other as that is more relevent and highlights how we tend to use factors - to code categorical data.  Perhaps add a sentence saying that are usually used for categorical variables.